cap-helm-charts-ci/tasks/create_pr.sh

```diff
@@ -62,6 +62,6 @@ pr_title="Release ${product}"
 pr_description="Publish Helm charts for release ${product} created from ${cap_bundle}."
 git pull
 git checkout -b ${product}
-git add stable/kubecf stable/cf-operator
+git add stable/kubecf stable/cf-operator stable/quarks
```

Comment:
Can you please move that into an environment variable and add that to the pipelines `.yaml`? 
It would be easier to adapt if we have that there.

Reply:
It would require a code change in the same repo or do I miss something?

Reply:
not much...
`git add stable/kubecf stable/cf-operator stable/quarks` -> `git add ${FOLDERS}`